tox.ini

```diff
@@ -58,6 +58,27 @@ commands =
     bash -euc "docker-compose run --rm web python manage.py test cabot || (docker-compose down && exit 1)"
     bash -euc "docker-compose down"
 
+# For running PostgreSQL-based tests multiple times:
```

Comment:
Added these two tox targets for (a) setting up psql tests, and (b) running them. Makes testing faster when running multiple times.